solutions/standard/variables.tf

```diff
@@ -61,6 +68,122 @@ variable "secret_manager_tags" {
   default     = []
 }
 
+variable "public_engine_enabled" {
+  type        = bool
+  description = "Set this to true to to configure an IBM Secrets Manager IAM credentials engine for an existing IBM Secrets Manager instance. If set to false, no iam engine will be configured for your secrets manager instance. For more details, see https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine."
+  default     = false
+}
+
+# Public cert engine config
+variable "public_engine_name" {
+  type        = string
+  description = "The name of the IAM Engine used to configure an IBM Secrets Manager IAM credentials engine for an existing IBM Secrets Manager instance."
+  default     = "public-engine-sm"
+}
+
+variable "cis_id" {
+  type        = string
+  description = "Cloud Internet Service ID"
```

Comment:
this has no default value so its makes it a required input, however its would not be required if not deploying the cert engine right?